README.md

```diff
@@ -10,17 +10,19 @@ Neural building blocks for speaker diarization:
 ## Installation
 
 ```bash
-# create a conda environment with Python 3.6 or later
-$ conda create --name pyannote python=3.6
+# Step 1: This creates a conda environment with python3.6
+$ conda create env -f environment.yml
```

Comment:
Just realized that this will fail because the repo has not been cloned yet.
Can you change the order?

Reply:
We need to talk about that, not sure about the whole ordeal

Reply:
Ok found the mistake, it should be 
```
conda env create -f environment.yml
```
(env and create should be inverted)

Reply:
I'm going to ask Marvin what's the right repo/branch to push to. It's getting confusing.